zio-http/src/main/scala/zhttp/http/Cookie.scala

```diff
@@ -12,7 +13,7 @@ import scala.util.Try
 final case class Cookie(
   name: String,
   content: String,
-  expires: Option[Instant] = None,
+  expires: Option[String] = None,
```

Comment:
We should keep time as an `Instant` only. The memory usage is much lower and modification is easier and doesn't require us to parse it multiple times.

Reply:
Sounds good! Took another shot at this. Still not able to figure out this one unit test issue though.